docs/dictionary/function/atan.lcdoc

```diff
@@ -33,7 +33,12 @@ The <atan> <function> <return|returns> a number between -pi/2 and pi/2.
 The result:
 The result of the <atan> function is <return|returned> in
 <radian|radians>. To get this result in <degree|degrees>, use the
-following <custom function>:.
+following <custom function>:
+
+    function atanInDegrees theMagnitude
+    return atan(theMagnitude) * 180 / pi
+
+    end atanInDegrees
```

Comment:
Could you make `theMagnitude` `pMagnitude` here and fix the indentation/blank line please?